Project.toml

```diff
@@ -6,7 +6,9 @@ version = "0.17.3"
 [deps]
 ArrayLayouts = "4c555306-a7a7-4459-81d9-ec55ddd5c99a"
 BandedMatrices = "aae01518-5342-5314-be14-df237901396f"
+CairoMakie = "13f3f980-e62b-5c42-98c6-ff1f3baf88f0"
```

Comment:
CairoMakie shouldn't be needed, and we don't want to add such a heavy dependency.

Reply:
Oops, added accidentally during a test run of the interpolation.jl example. Good catch.